Makefile.js

```diff
@@ -705,7 +723,16 @@ target.gensite = function(prereleaseVersion) {
                 } else {
                     title = "Documentation";
                 }
-                text = `---\ntitle: ${title}\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n\n${text}`;
+                text = [
+                    "---",
+                    `title: ${title}`,
+                    "layout: doc",
+                    `edit_link: https://github.com/eslint/eslint/edit/master/${baseName}`,
```

Comment:
Should this use `filename` instead of `baseName`? Otherwise, I think a file like `docs/about/index.md` will map to a link like `https://github.com/eslint/eslint/edit/master/index.md`, resulting in a 404.